src/rez/utils/platform_.py

```diff
@@ -482,7 +482,7 @@ def symlink(self, source, link_name):
                 raise ctypes.WinError()
 
     def _terminal_emulator_command(self):
-        return "CMD.exe /Q /K"
+        return "START"
```

Comment:
"terminal emulator" is probably a bit misleading.  This command is only used when launching a rez environment using `detached=True`.  `CMD.exe` just spawns a subshell within the current shell.  `START` will launch a separate shell.